types/jquery/JQuery.d.ts

```diff
@@ -6167,6 +6167,81 @@ $( "p span" ).last().addClass( "highlight" );
 ```
      */
     last(): this;
+
+    /**
+     * Reduce the set of matched elements to the even ones in the set, numbered from zero.
+     * @see \`{@link https://api.jquery.com/even/ }\`
+     * @since 3.5
+     * @example ​ ````Highlight the even items in a list.
+```html
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>even demo</title>
+  <style>
+  .highlight {
+    background-color: yellow;
+  }
+  </style>
+  <script src="https://code.jquery.com/jquery-3.5.0.js"></script>
+</head>
+<body>
+
+<ul>
+  <li>Look:</li>
+  <li>This is some text in a list.</li>
+  <li>This is a note about it.</li>
+  <li>This is another note about it.</li>
+</ul>
+
+<script>
+$( "ul li" ).even().addClass( "highlight" );
+</script>
+
+</body>
+</html>
+```
+     */
+    even(): this;
+
+    /**
+     * Reduce the set of matched elements to the odd ones in the set, numbered from zero.
+     * @see \`{@link https://api.jquery.com/odd/ }\`
+     * @since 3.5
+     * @example ​ ````Highlight the odd items in a list.
+```html
```

Comment:
![HTML example rendered in tooltip](https://user-images.githubusercontent.com/5265970/85231729-a23d1f80-b3c7-11ea-8630-840013de5931.png)

The HTML example will render in tooltips. Having the example in-editor reduces friction for developers who'd otherwise have to open the docs in a browser and then switch back to continue working.

Reply:
I know but I still prefer documentation opened in a browser because there will always exist actual documentation.

Reply:
@leonard-thieu @martin-badin thx for the review!